samples/bluetooth/mesh/light_ctrl/src/model_handler.c

```diff
@@ -21,6 +21,37 @@ struct lightness_ctx {
 	uint32_t rem_time;
 };
 
+#if IS_ENABLED(CONFIG_BT_MESH_COMP_PAGE_2)
+static const uint8_t cmp2_elem_offset1[2] = { 0, 1 };
+static const uint8_t cmp2_elem_offset2[1] = { 0 };
+
+static const struct bt_mesh_comp2_record comp_rec[2] = {
+	{
+	.id = 0x1601, /* Basic Lightness Controller NLC Profile 1.0 */
```

Comment:
Please define macros for these values, as they are assigned numbers. May be in `models.h`.

Reply:
We do not have any defined macros for these model ids.

Reply:
I'm suggesting to define new ones in `models.h`.